subsys/net/lib/lwm2m/lwm2m_obj_security.c

```diff
@@ -72,7 +96,14 @@ static struct lwm2m_engine_obj_field fields[] = {
 	OBJ_FIELD_DATA(SECURITY_LWM2M_SERVER_SMS_NUM_ID, W_OPT, STRING),
 	OBJ_FIELD_DATA(SECURITY_SHORT_SERVER_ID, W_OPT, U16),
 	OBJ_FIELD_DATA(SECURITY_CLIENT_HOLD_OFF_TIME_ID, W_OPT, U32),
-	OBJ_FIELD_DATA(SECURITY_BS_SERVER_ACCOUNT_TIMEOUT_ID, W_OPT, U32)
+	OBJ_FIELD_DATA(SECURITY_BS_SERVER_ACCOUNT_TIMEOUT_ID, W_OPT, U32),
```

Comment:
I don't know does those really matter but the following fields are defined as signed integers - S32s - by the specification

![image](https://user-images.githubusercontent.com/33285290/147248187-ee86292d-9143-4f3d-9aed-02eb0e50a7a7.png)

Reply:
Good catch. First of those 2 are already in the version 1.0, but have probably seen no use as they aren't initialized at all in the object instance creation.